04-networking.md

```diff
@@ -85,7 +85,7 @@ The following two resource groups will be created and populated with networking
    The spoke creation will emit the following:
 
      * `appGwPublicIpAddress` - The Public IP address of the Azure Application Gateway (WAF) that will receive traffic for your workload.
-     * `clusterVnetResourceId` - The resource ID of the VNet that the cluster will land in. E.g. `/subscriptions/[id]/resourceGroups/rg-enterprise-networking-spokes/providers/Microsoft.Network/virtualNetworks/vnet-spoke-BU0001A0008-00`
+     * `clusterVnetResourceId` - The resource ID of the Virtual network that the cluster, App Gateway, and related will be deployed in. E.g. `/subscriptions/[id]/resourceGroups/rg-enterprise-networking-spokes/providers/Microsoft.Network/virtualNetworks/vnet-spoke-BU0001A0008-00`
```

Comment:
Are you missing a noun after related? The related what will be deployed?

Reply:
Good catch, update will be coming shortly.

Reply:
Resolved in: https://github.com/mspnp/aks-baseline/pull/326/commits/3f3058684316dffe84b7b6180ccbf5a212c27a94